src/modules/flow/platform/platform.json

```diff
@@ -36,6 +36,22 @@
       "url": "http://solettaproject.org/doc/latest/node_types/platform.html"
     },
     {
+      "category": "input/sw",
+      "description": "Node used to set the platform target",
+      "in_ports": [
+        {
+          "data_type": "string",
+          "description": "The new platform target to set which must be one of: default, rescue, emergency, poweroff, reboot or suspend.",
+          "methods": {
+            "process": "platform_target_process"
+          },
+          "name": "TARGET"
+        }
+      ],
+      "name": "platform/target",
```

Comment:
call it 'set-target' or make it provide an output port to list the current target ... not sure there is such thing.

Also provide a sample on its usage. A common pattern is good to handle: press-and-hold GPIO for 10s to reboot. If one releases early, the timer should be cancelled.